cmd/buffer-api/main.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"context"
+	"flag"
+	stdLog "log"
+	"net"
+	"net/url"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/api/server/buffer/dependencies"
+	bufferGen "github.com/keboola/keboola-as-code/internal/pkg/api/server/buffer/gen/buffer"
+	bufferHttp "github.com/keboola/keboola-as-code/internal/pkg/api/server/buffer/http"
+	"github.com/keboola/keboola-as-code/internal/pkg/api/server/buffer/service"
+	"github.com/keboola/keboola-as-code/internal/pkg/env"
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/pkg/errors"
+)
+
+type ddLogger struct {
+	*stdLog.Logger
+}
+
+func (l ddLogger) Log(msg string) {
+	l.Logger.Print(msg)
+}
+
+func main() {
```

Comment:
Tady entrypoint, jediný rozdíl od templates je to, že tu není všechno okolo template repositories.